internal/service/privateendpointregionalmode/resource_private_endpoint_regional_mode_test.go

```diff
@@ -179,32 +176,6 @@ func checkExists(resourceName string) resource.TestCheckFunc {
 	}
 }
 
-func checkModeClustersUpToDate(projectID, clusterName, clusterResourceName string) resource.TestCheckFunc {
```

Comment:
not useful?

Reply:
Super verose and fails to report error when no PrivateEndpoint is set. (Only compares to whatever the sdk returns for the cluster)